### PR TITLE
Update addressbook to keep track of student, tutors and classes

### PR DESCRIPTION
**Pull request by ongwenpin**

### CODE REVIEW

File: src/main/java/seedu/address/model/AddressBook.java [hunk 1]
```diff
@@ -56,57 +59,153 @@ public void setPersons(List<Person> persons) {
         this.persons.setPersons(persons);
     }
 
+    /**
+     * Replaces the contents of the student list with {@code students}.
+     * {@code students} must not contain duplicate students.
+     */
+    public void setStudents(List<Person> students) {
```
Comment (tantzeyeong) [inline src/main/java/seedu/address/model/AddressBook.java:66]: Perhaps we should use wildcard here? i.e. List<? extends Person>

File: src/main/java/seedu/address/model/AddressBook.java [hunk 2]
```diff
@@ -56,57 +59,153 @@ public void setPersons(List<Person> persons) {
...
+        this.students.setPersons(students);
+    }
+
+    /**
+     * Replaces the contents of the tutor list with {@code tutors}.
+     * {@code tutors} must not contain duplicate tutors.
+     */
+    public void setTutors(List<Person> tutors) {
```
Comment (tantzeyeong) [inline src/main/java/seedu/address/model/AddressBook.java:74]: Same case as above

**Review by tantzeyeong — COMMENTED** [review]
Just a few changes to consider. Other than that, LGTM!

File: src/main/java/seedu/address/model/AddressBook.java (hunk 1 above)
Comment (ongwenpin) [reply src/main/java/seedu/address/model/AddressBook.java]: ok

**Comment by tantzeyeong** [comment]
LGTM!

**Pull request merged**